public-api/db-cdr/generate-cdr/run-achilles-queries.sh

```diff
@@ -1483,16 +1660,29 @@ bq --quiet --project=$BQ_PROJECT query --nouse_legacy_sql \
 (id,analysis_id,stratum_1,stratum_2,stratum_3,count_value,source_count_value)
 select 0, 3107 as analysis_id,
 CAST(sm.concept_id AS STRING) as stratum_1,
-CAST(p1.gender_identity_concept_id AS STRING) as stratum_2,'Survey' as stratum_3,
+CAST(p1.value_source_concept_id AS STRING) as stratum_2,'Survey' as stratum_3,
 COUNT(distinct p1.PERSON_ID) as count_value,COUNT(distinct p1.PERSON_ID) as source_count_value
-from \`${WORKBENCH_PROJECT}.${WORKBENCH_DATASET}.person_gender_identity\` p1 inner join
-\`${BQ_PROJECT}.${BQ_DATASET}.observation\` ob1
-on p1.person_id = ob1.person_id
+from \`${BQ_PROJECT}.${BQ_DATASET}.observation\` ob1 join \`${BQ_PROJECT}.${BQ_DATASET}.observation\` p1 on p1.person_id=ob1.person_id
 join \`${WORKBENCH_PROJECT}.${WORKBENCH_DATASET}.survey_question_map\` sq
 On ob1.observation_source_concept_id=sq.question_concept_id
 join \`${WORKBENCH_PROJECT}.${WORKBENCH_DATASET}.survey_module\` sm on sq.survey_concept_id = sm.concept_id
-where (ob1.observation_source_concept_id > 0 and ob1.value_source_concept_id > 0)
-group by sm.concept_id, p1.gender_identity_concept_id"
+where (ob1.observation_source_concept_id > 0 and ob1.value_source_concept_id > 0) and p1.observation_source_concept_id=1585838
+group by sm.concept_id, p1.value_source_concept_id"
+
+# Race ethnicity breakdown of people who took each survey (Row for combinations of each survey and gender)
+bq --quiet --project=$BQ_PROJECT query --nouse_legacy_sql \
```

Comment:
Is there a ticket to refactor this script - if not, do you mind filing one? There's a lot of duplication here so it's pretty hard to review in its current state.

Reply:
https://precisionmedicineinitiative.atlassian.net/browse/DB-362